tests/test050.cpp

```diff
@@ -0,0 +1,36 @@
+// test003.cpp - read column header by index
```

Comment:
Thanks for the feedback on this PR!!! Apologies for the quality.

Just to make sure, should I implement GetRowLabel? I don't see it in the source

Thanks again

Reply:
Well, thank you for taking the time to improve this library!
No, let's wait with GetRowLabel, I realized that it's an odd use-case, so there's no point adding it just for sake of consistency. Once someone needs it, it can be added.